packages/ui/src/components/notifications/NotificationCenter.tsx

```diff
@@ -0,0 +1,88 @@
+import { ComponentType, useEffect, useState } from 'react';
+
+import { Notification } from './types';
+
+const NOTIFICATION_EVENT = 'emit-core-notification';
+const DEFAULT_CACHE_TIMEOUT = 5 * 60 * 1000; // 5 minutes in milliseconds
```

Comment:
if you call this `DEFAULT_CACHE_TIMEOUT_MS` it would be pretty clear that there are  5 minutes there from  ` 5 * 60 * 1000` and we won't need the comment